aea/cli/utils/config.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""A module with config tools of the aea cli."""
+
+import logging
+import logging.config
+import os
+from pathlib import Path
+from typing import Dict
+
+import click
+
+import jsonschema  # type: ignore
+
+import yaml
+
+from aea.cli.utils.constants import CLI_CONFIG_PATH
+from aea.cli.utils.context import Context
+from aea.cli.utils.package_utils import _load_yaml
+from aea.configurations.base import DEFAULT_AEA_CONFIG_FILE
+
+
+def try_to_load_agent_config(
```

Comment:
seems a prime candidate for making this a helper method on the `Context` object in `../utils/context.py` rather than a helper function.

Reply:
Also worth to do, totally agree. But too many code rewriting for this PR. I'll have to manually re-write an every usage of those methods (and that are the most commonly used funcs). That take a lot of time and line changes.